vips/image_test.go

```diff
@@ -242,11 +262,36 @@ func TestImageRef_RemoveOrientation__NoEXIF(t *testing.T) {
 	require.NoError(t, err)
 	defer image.Close()
 
-	image.RemoveOrientation()
+	err = image.RemoveOrientation()
+	require.NoError(t, err)
+
+	assert.Equal(t, 0, image.GetOrientation())
+}
+
+func TestImageRef_RemoveMetadata(t *testing.T) {
```

Comment:
please rename and add another part that specifies the expected outcome

Reply:
not a great fan of lengthy func names, especially in simple cases such as this